ocs_ci/ocs/bucket_utils.py

```diff
@@ -63,6 +68,7 @@ def craft_s3_command(cmd, mcg_obj=None, api=False, signed_request_creds=None):
             f"{region}"
             f"aws s3{api} "
             f'--endpoint={signed_request_creds.get("endpoint")} '
+            f"{no_ssl} "
```

Comment:
f"{no_ssl}"

Reply:
The whitespace doesn't really do any harm in its current state, and can be convenient in case we'd like to expand the command further in the future